src/_node-compat/headers.ts

```diff
@@ -279,8 +279,8 @@ function _normalizeValue(
 }
 
 function validateHeader(name: string): string {
-  if (name[0] === ":" || name.includes(":")) {
-    throw new TypeError("Invalid name");
+  if (name[0] === ":") {
```

Comment:
I don't think that's correct. Tests could create a header like `x-app:test=1`, which is also invalid.

Reply:
@pi0 ☝️

Reply:
tests was wrongly creating it because input (request headers) was feeding in invalid `:test` header. in between checks add overhead (full string chars should be checked)